Makefile

```diff
@@ -67,7 +67,7 @@ install-dom0: all-dom0
 
 
 all-vm:
-	+$(MAKE) all -C agent
+	+$(MAKE) all CFLAGS=-DHAVE_PAM LDLIBS=-lpam -C agent
```

Comment:
You override CFLAGS/LDLIBS here, which breaks reproducible builds (drops flags set by dpkg-buildpackage, rpmbuild etc).
But also, setting LDLIBS in this place looks wrong (see the next comment).